bundle/src/main/groovy/org/cid15/aem/groovy/console/constants/GroovyConsoleConstants.groovy

```diff
@@ -5,11 +5,13 @@ import com.google.common.net.MediaType
 
 class GroovyConsoleConstants {
 
-    public static final String SYSTEM_USER_NAME = "groovy-console-system-user"
+    public static final String SYSTEM_USER_NAME = "aem-groovy-console-service"
 
     public static final String PATH_CONSOLE_ROOT = "/var/groovyconsole"
```

Comment:
where is the console root still used? should it still be under /var?

Reply:
for the audit nodes

Reply:
to be discussed if we want to move

Reply:
No that's ok, those are generated nodes, not nodes that are replicated